workflows/mag.nf

```diff
@@ -801,7 +813,13 @@ workflow MAG {
             }
 
             BINNING_REFINEMENT ( ch_contigs_for_binrefinement, ch_prokarya_bins_dastool )
-            ch_refined_bins = ch_eukarya_bins_dastool.mix(BINNING_REFINEMENT.out.refined_bins)
+            // ch_refined_bins = ch_eukarya_bins_dastool
+            //     .map{ meta, bins ->
+            //             def meta_new = meta + [refinement: 'eukaryote_unrefined']
+            //             [meta_new, bins]
+            //         }.mix( BINNING_REFINEMENT.out.refined_bins)
```

Comment:
Why is commented out code added?

Reply:
See [above](https://github.com/nf-core/mag/pull/541#discussion_r1390937452)